styles.css

```diff
@@ -0,0 +1,44 @@
+#body{
+    font-family: Arial, Helvetica, sans-serif;
+   
+    color:white;
+    margin: 0;
+    background-image: linear-gradient( 115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7) ), url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
```

Comment:
- I think it would be better to check the background image from this section because now it is not been displayed.